pkg/managed/deployment.go

```diff
@@ -96,10 +100,14 @@ func newDeployments(binding *v1beta1v8o.VerrazzanoBinding, managedCluster *types
 	// Does a Prometheus pusher need to be deployed to this cluster?
 	if deployPromPusher == true {
 		if verrazzanoURI == "" {
-			glog.V(4).Infof("Verrazzano URI must not be empty for prometheus pusher to work")
+			glog.Error("Verrazzano URI must not be empty for prometheus pusher to work")
```

Comment:
So this error is simply logged & ignored?  I realize all you did was change the loglevel but if this is truly an error I'd think it should be returned.

Reply:
That is sort of beyond the scope of monitoring/deployment but i can do that and change the code that uses this function.